src/fragments/ui/auth/flutter/authenticator.mdx

```diff
@@ -0,0 +1,335 @@
+## Basic Integration
+
+The Amplify-Flutter Authenticator Widget is a Material-compatible sign-up and sign-in experience which reacts to your Amplify configuration. It automatically detects the authentication state and updates the UI. If the user is signed in, the underlying component (typically your app's home widget) is displayed; otherwise, signin/signup controls are displayed.
+
+To begin, follow the steps for configuring the Amplify Auth category from the [getting started walkthrough](/lib/auth/getting-started). Be sure to also add the Amplify-Flutter Authenticator to your `pubspec.yaml` file:
+
+```yaml
+dependencies:
+  amplify_authenticator: ^1.0.0-rc.0
+```
+
+You will also need to import the Amplify-Flutter Authenticator into your app:
+
+```dart
+import 'package:amplify_authenticator/amplify_authenticator.dart';
```

Comment:
Import `amplify_flutter` and `amplify_auth_cognito` as well

Reply:
Need to mention calling `Amplify.configure`, too, I would think

Reply:
I didn't want to duplicate too much content that was already mentioned in the getting started. What about a link to that section?

Reply:
Either way - but in case they end up here without having seen that, they may struggle to get things working correctly

Reply:
@dnys1 Ok - I'm making line 5 more explicit:
`To begin, be sure to follow the steps for importing and configuring Amplify and the Amplify Auth category from the [getting started walkthrough](/lib/auth/getting-started).`